client/src/Ranger.js

```diff
@@ -0,0 +1,16 @@
+import React, {Component} from "react";
+
+class Ranger extends Component {
```

Comment:
Same thing I mentioned on Cameron's PR, since React is moving away from class components and more towards functional components, I think we should refactor this to be a functional component

Reply:
This would be very straightforward here, we would change it to:
```
const Ranger = () => {
   return ( ... );
}
```
or
```
function Ranger() {
  return ( ... );
}
```

Reply:
Updated to use functional!